js_modules/dagit/packages/core/src/workspace/useRepositoryForRun.ts

```diff
@@ -48,7 +48,7 @@ export const useRepositoryForRun = (
     }
 
     const pipelineName = run.pipelineName;
-    const snapshotId = run.pipelineSnapshotId;
+    const snapshotId = run.parentPipelineSnapshotId ?? run.pipelineSnapshotId;
```

Comment:
Is it always safe just to check one snapshot ID or the other, or is it possible we may want to check both?

Reply:
My understanding is that the pipelineSnapshotId found in the repository (i.e. what we're checking against) will always be the parentPipelineSnapshotId if there is a parent PipelineSnapshot. If there's no parent, that means the job wasn't subsetted, and so we'll compare against the regular pipelineSnapshotId

So I think this should be ok